### PR TITLE
add cmdLineTester_CryptoTest for OpenSSL

### PR DESCRIPTION
**Pull request by llxia**
Issue: #4092

[ci skip]

Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
Passed on:
JDK8 xlinux: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/210/
JDK8 win32: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/211/
JDK8 win64: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/212/
JDK8 ppcle: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/214/

Failed on:
JDK8 osx: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/213/console
```
00:07:01.057 Output from test:
00:07:01.057  [OUT] Crypto test COMPLETED
00:07:01.057  [ERR] Warning: Native crypto library load failed. Using Java crypto implementation
00:07:01.057 >> Success condition was not found: [Output match: using Native crypto library]
00:07:01.057 >> Required condition was found: [Output match: Crypto test COMPLETED]
00:07:01.057 >> Failure condition was found: [Output match: Native crypto library load failed]
00:07:01.057 >> Failure condition was not found: [Output match: Crypto test FAILED]
```
@pshipton Is this a known issue? should I open an issue for the failure on osx?

**Comment by llxia** [comment]
Passed on:
JDK11 xlinux: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/215/
JDK11 ppcle: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/217/
JDK11 s390: 
JDK11 xlinux LH: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/221/

Failed on JDK11 osx: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/220/tapResults/
Failed on JDK11 win64: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/222/tapResults/

**Comment by pshipton** [comment]
For Windows jdk11, I see the jncrypto.dll but not a libcrypto-1_1-x64.dll so it seems the Adopt build is missing the `--enable-openssl-bundling` configuration option.

**Comment by pshipton** [comment]
For osx jdk11 I see both libjncrypto.dylib and libcrypto.1.1.dylib present. Seems an extensions problem to debug why it is not working.

**Comment by pshipton** [comment]
Please open issues for both problems on the extensions repo.

For Windows, can confirm that OpenJ9 builds with `--enable-openssl-bundling`, however the crypto dll doesn't appear to be included in the build. I see it in the jdk8 build but not jdk11.
```
00:45:47 + bash configure --with-freemarker-jar=/cygdrive/c/openjdk/freemarker.jar --with-boot-jdk=/cygdrive/c/openjdk/jdk11 --with-toolchain-version=2017 --disable-ccache --with-openssl=/cygdrive/c/OpenSSL-1.1.1-x86_64 --enable-openssl-bundling```

**Comment by llxia** [comment]
Issue created: https://github.com/ibmruntimes/openj9-openjdk-jdk11/issues/98

**Comment by pshipton** [comment]
@llxia please exclude the failing platforms so the test can be delivered.

**Comment by llxia** [comment]
- The test is excluded on aix and osx for both jdk8 and jdk11.
Grinder:
The test is excluded on jdk11 osx: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/234/
The test is excluded on jdk8 osx: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/232/
The test is excluded on jdk8 aix: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/237/
The test is excluded on jdk11 aix: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/238/

- The test is also excluded on jdk11 win.
Grinder:
The test is excluded on jdk11 win: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/235/
The test still runs on win jdk8: https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/236/

**Comment by llxia** [comment]
Ready for review.

File: test/functional/cmdLineTests/openssl/openssl.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+
+<!--
+  Copyright (c) 2013, 2019 IBM Corp. and others
+
```
Comment (smlambert) [inline test/functional/cmdLineTests/openssl/openssl.xml]: If its a new file, shift 2013 to 2019.

Comment (llxia) [reply test/functional/cmdLineTests/openssl/openssl.xml]: ah, thanks. Updated.

**Comment by smlambert** [comment]
I will merge without PR testing as Grinders above were thorough.

**Comment by pshipton** [comment]
@llxia can you please create a PR with the same changes for the v0.12.0-release branch as well.

**Pull request merged**